content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: A New Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Users
+description: A new deployment model for Istio.
+publishdate: 2020-08-24
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,central istiod,install,deploy,'1.7']
+---
+
+## Background
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+
+* Mesh Operator, who manages the service mesh installation and upgrade.
+
+* Mesh Admin/Users:
+
+  1. Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
+  1. Service Owner, who owns one or more services.
+
+It is common to have all these personas work on the same clusters without any clear separation.  For example, there are multiple ways to deploy Istio according to the [docs](/docs/setup/install/), which all start with mesh operator, platform owner and service owner sharing the single cluster first and then gradually expanding the mesh to multiple clusters or VMs.  None of these provided a clear separation between mesh operator and platform/service owner at the boundary of a cluster.  You may be thinking you could set up [Kubernetes RBAC rules](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) and namespaces to control which personas can do what within the cluster, however, sometimes you need stronger separation between the Istio control plane and the rest at the cluster level.  We're introducing a new deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on dedicated clusters, separated from the data plane clusters.  This new deployment model could easily enable Istio vendors to run Istio control plane for service mesh users while users can focus on their workloads and Istio resources without worrying about installing or managing the Istio control plane. This model enables vendors to build managed Istio offerings that automatically upgrade Istio control plane for mesh users.
+
+## Central Cluster Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you can deploy your services to the mesh, like the diagram below:
```

Comment:
I would say something about how the default installation of Istio uses a single cluster for the control plane and the services, and then show the diagram. I wouldn't talk about deploying services since it isn't really relevant.

Reply:
Updated.  I talked about deploying services as the diagram had services.  But I agree, there is no change to that experience so not worthy highlighting.